api-report/odsp-driver.api.md

```diff
@@ -175,38 +172,13 @@ export interface OdspFluidDataStoreLocator extends IOdspUrlParts {
 }
 
 // @public
-export function parseCompactSnapshotResponse(buffer: ReadBuffer): ISnapshotContents;
+export function parseCompactSnapshotResponse(buffer: Uint8Array, logger: ITelemetryLogger): ISnapshotContents;
```

Comment:
@anthony-murphy, you might want to take a look here (or assign to someone else). This change did not trigger build break and requirement to add FunctionDeclaration_parseCompactSnapshotResponse to package.json.